.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+defaults: &DEFAULT
+  working_directory: /home/circleci/xblock-html
+  docker:
+    - image: circleci/python:2.7
+
+
+version: 2
+jobs:
+  build:
+    <<: *DEFAULT
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps1-{{ .Branch }}-{{ checksum "requirements/circle.txt" }}
+      - run:
+          name: Installing requirements
+          command: |
+            virtualenv env
```

Comment:
Can you move the env out of the source dir? See https://github.com/edx-olive-oc/edx-platform/commit/e5a883e3c5024cc4a5c52a7b9b94d7bb44044589#diff-1d37e48f9ceff6d8030570cd36286a61